docs/usage/self-hosted-configuration.md

```diff
@@ -626,6 +626,8 @@ Set this to `"enabled"` to have Renovate maintain a JSON file cache per-reposito
 Set to `"reset"` if you ever need to bypass the cache and have it overwritten.
 JSON files will be stored inside the `cacheDir` beside the existing file-based package cache.
 
+## repositoryCacheType
```

Comment:
missing description ?

Reply:
no, when auto generated the description from `lib/config/options/index.ts` is appended. i just dont have much to add apart from what is already written there